### PR TITLE
Evaluation system

### PR DESCRIPTION
**Pull request by LauraMQuiros**
- word overlap calculation 
- perplexity score calculation
- BERTScore calculation 
- both a python and notebook file for easy habrok evaluation 
- approx 1h for test dataset generation
- missing the classifier but this is the rest so far

### CODE REVIEW

File: tests/eval_model.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        data = pickle.load(f)
+
+    model_predictions = get_model_predictions(lora_model, lora_tokenizer, data)
+
+    # Give the fluency of the model (perplexity score with Kneser-Ney smoothing)
+    PPL_score = get_perplexity_kneser_ney(model_predictions)
+    print(f'The perplexity of the model is: {PPL_score}')
+
+    # Give the content-preservation score of the model (BERT score and word overlap)
+    # BERT_score = get_BERT_score(model_predictions, data)
```
Comment (Pempoczky) [inline tests/eval_model.py:130]: why was this commented out?

**Review by Pempoczky — APPROVED** [review]
Looks good to me, though I see the BERT score is not working yet. Why is that?

**Review by YoKo28 — APPROVED** [review]
looks good, bert score needs to be fixed

**Review by YoKo28 — APPROVED** [review]
looks good

**Pull request merged**